aquadoggo/src/schema_service/schema_service.rs

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+// @TODO: The API changed here in `p2panda_rs`
```

Comment:
:point_up: This whole thing gets replaced by the `SchemaProvider`, which is developed here: https://github.com/p2panda/aquadoggo/pull/166 - so no need to worry too much about this file.